caikit/runtime/http_server/http_server.py

```diff
@@ -158,7 +158,10 @@ async def request_validation_exception_handler(
                 "id": uuid.uuid4().hex,
             }
             log.error("<RUN59871106E>", error_content, exc_info=True)
-            return Response(content=json.dumps(error_content), status_code=err_code)
+            return Response(
+                content=json.dumps(jsonable_encoder(error_content)),
+                status_code=err_code,
```

Comment:
We already use `jsonable_encoder` in the response validation! Worth investigating what the difference is between `Response` and `JSONResponse`. We can think about returning the same in both validations

Reply:
Also we should just be able to remove `json.dumps` altogether?
```
(content=jsonable_encoder(error_content), status_code=err_code)
```

Reply:
Using `jsonable_encoder` returns the dict, that no longer has the byte object, but still the bad json
so using the `json.dumps` converts it to `str` so it can be passed without error

Reply:
The `JSONResponse` does not get called when I put a breakpoint in the `validation_exception_handler` method. Looking into the call stack to figure out how / when that is called?

Reply:
Yeah it's hard to trigger the response validation since I don't think we have a test for that. Better to read up on the implementation differences between the 2.